### PR TITLE
enable checkJs for many packages

### PR DESCRIPTION
**Pull request by turadg**
refs: #4885 (partial completion)

## Description

For key packages, flip the _opt-in_ `ts-check` to be opt-out and check all .js files by default. 

I expect we'll want to break this up into other PRs so I'm leaving this one as Draft. I have them all gathered to see how they interplay. The commits also depend on each other somewhat so it provides an ordering for landing.

Note that many of the types are still ambient. But the removal of ts-check reduces costs to package consumers, like https://github.com/Agoric/wallet-app/commit/7cad8073b4529ced764a1809351c58bc430536a2

### CODE REVIEW

**Comment by Chris-Hibbert** [comment]
I looked at governance, inter-protocol, and zoe, and those changes would be easy to approve.

File: packages/SwingSet/src/devices/lib/deviceTools.js [hunk 1]
```diff
@@ -69,6 +69,7 @@ export function buildSerializationTools(syscall, deviceName) {
     if (devnodeSlot) {
       return devnodeSlot;
     }
+    // @ts-expect-error DetailsToken not a string
     throw Error(X`unable to convert value ${val}`);
```
Comment (erights) [inline packages/SwingSet/src/devices/lib/deviceTools.js:73]: See https://github.com/Agoric/agoric-sdk/pull/6538

Comment (erights) [reply packages/SwingSet/src/devices/lib/deviceTools.js:73]: Btw, it is a victory of this PR that it triggered this ts error! This is the opportune time to notice whether such newly triggered diagnostics indicate a genuine problem that we previously missed.

**Comment by turadg** [comment]
All changes here have landed through other PRs

**Pull request closed without merging**